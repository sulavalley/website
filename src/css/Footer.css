.footer {
  position: fixed;
  left: 0;
  bottom: 0;
  height: 75px;
  width: 100%;
  background-color: #3399ff;
  color: white;
  text-align: center;
}

.margin-info {
  margin-top: 10px;
}
