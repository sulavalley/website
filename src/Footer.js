import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <div className="footer">
      <div className="margin-info">
        <h4>&copy; Copyright 2017 | All rights reserved</h4>
      </div>
    </div>
  );
};

export default Footer;
