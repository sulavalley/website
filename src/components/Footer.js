import React from 'react';
import './../css/Footer.css';

const Footer = () => {
  return (
    <div className="footer">
      <div className="margin-info">
        <h4>
          Página desarrollada por Andrés Rodríguez :D | Mesa regional de turismo
          del Valle de Sula
        </h4>
      </div>
    </div>
  );
};

export default Footer;
