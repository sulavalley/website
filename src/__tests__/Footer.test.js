import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Footer from './../components/Footer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<Footer />);
});

test('renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
