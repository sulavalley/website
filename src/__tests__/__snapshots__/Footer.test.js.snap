// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="margin-info"
  >
    <h4>
      Â© Copyright 2017 | All rights reserved
    </h4>
  </div>
</div>
`;
